src/transitions/scroll-transition.js

```diff
@@ -140,16 +140,17 @@ export default class ScrollTransition extends React.Component {
       }
     }
 
-    return targetPosition + offset;
+    return targetPosition + offset || 0;
```

Comment:
Added a default return value of 0.  It looks like in some of our tests (with things not fully configured) it was returning `NaN` which was giving us a warning.  I figure if it calculates to `NaN` we can just send back 0 so things don't break (and again, it seems this only occurred in our tests).